civicpy/civic.py

```diff
@@ -3193,3 +3380,23 @@ def get_phenotype_by_name(name):
     if len(matching_phenotypes) == 0:
         raise Exception("No phenotypes with name: {}".format(name))
     return matching_phenotypes[0]
+
+# Endorsement
+
+def search_endorsements_by_organization_id(organization_id):
+    """
+    :param int organization_id: A CIViC :class:`Organization` ID.
+    :returns: A list of :class:`Endorsement` objects.
+    """
+    endorsements = _get_elements_by_ids('endorsement', get_all=True)
+    matching_endorsements = [e for e in endorsements if e.organization_id == organization_id]
+    return endorsements
+
+def search_endorsements_by_assertion_id(assertion_id):
+    """
+    :param int assertion_id: A CIViC :class:`Assertion` ID.
+    :returns: A list of :class:`Endorsement` objects.
+    """
+    endorsements = _get_elements_by_ids('endorsement', get_all=True)
+    matching_endorsements = [e for e in endorsements if e.assertion_id == assertion_id]
+    return endorsements
```

Comment:
@susannasiebert I think there was a typo

```suggestion
def search_endorsements_by_organization_id(organization_id):
    """
    :param int organization_id: A CIViC :class:`Organization` ID.
    :returns: A list of :class:`Endorsement` objects.
    """
    endorsements = _get_elements_by_ids('endorsement', get_all=True)
    matching_endorsements = [e for e in endorsements if e.organization_id == organization_id]
    return matching_endorsements

def search_endorsements_by_assertion_id(assertion_id):
    """
    :param int assertion_id: A CIViC :class:`Assertion` ID.
    :returns: A list of :class:`Endorsement` objects.
    """
    endorsements = _get_elements_by_ids('endorsement', get_all=True)
    matching_endorsements = [e for e in endorsements if e.assertion_id == assertion_id]
    return matching_endorsements
```